Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:16
+
+LABEL maintainer="me@jqshuv.xyz"
+LABEL org.opencontainers.image.source https://github.com/ZynerOrg/xyter
+LABEL org.opencontainers.image.description "An multi-purpose discord.js bot."
+LABEL org.opencontainers.image.licenses=GPL-3.0-only
+
+
+WORKDIR /app
+
+LABEL maintainer="me@jqshuv.xyz"
+LABEL org.opencontainers.image.source https://github.com/ZynerOrg/xyter
+LABEL org.opencontainers.image.description "An multi-purpose discord.js bot."
+LABEL org.opencontainers.image.licenses=GPL-3.0-only
+
+COPY package* .
+RUN npm install
+
+COPY . .
```

Comment:
Is this really necessary? You are copying from current dir to current dir., right?

Reply:
No, I copy from the current local directory to the docker workdir.

Reply:
Oh I see!